<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>学生学籍管理系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Create_StuInfo_Button">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>102</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>创建学生信息列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="isCreate_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>116</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>尚未创建打开学生信息列表，无法正常使用系统功能！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_StuInfo_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>添加学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_StuInfo_Button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>删除学生信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search_StuInfo_Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>查询学生信息</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>751</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>按学号排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按微积分1-1成绩排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按微积分1-2成绩排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按C语言程序成绩排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按数据结构成绩排序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按绩点排序</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Sort_Button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
